packages/cli/bin/cli.js

```diff
@@ -51,12 +52,30 @@ const notifier = updateNotifier({
 
 const i18nKey = 'commands.generalErrors';
 
-const CLI_UPGRADE_MESSAGE =
-  chalk.bold('The CMS CLI is now the HubSpot CLI') +
-  '\n\nTo upgrade, run:\n\nnpm uninstall -g @hubspot/cms-cli\nand npm install -g @hubspot/cli';
+const CMS_CLI_PACKAGE_NAME = '@hubspot/cms-cli';
 
 notifier.notify({
-  message: pkg.name === '@hubspot/cms-cli' ? CLI_UPGRADE_MESSAGE : null,
+  message:
+    pkg.name === CMS_CLI_PACKAGE_NAME
+      ? i18n(`${i18nKey}.updateNotify.cmsUpdateNotification`, {
+          packageName: CMS_CLI_PACKAGE_NAME,
+          updateCommand: uiCommandReference('{updateCommand}'),
```

Comment:
Just for posterity, I'm passing in `{updateCommand}` here instead of the actual command because update-notify handles some interpolation for certain strings. They mention them in the docs [here](https://www.npmjs.com/package/update-notifier#message).